lib/agent.js

```diff
@@ -556,9 +558,9 @@ Agent.prototype.handleUncaughtExceptions = function (cb) {
   this._uncaughtExceptionListener = function (err) {
     agent.logger.debug({ err }, 'Elastic APM caught unhandled exception')
     // The stack trace of uncaught exceptions are normally written to STDERR.
-    // The `uncaughtException` listener inhibits this behavor, and it's
+    // The `uncaughtException` listener inhibits this behavior, and it's
     // therefore necessary to manually do this to not break expectations.
-    if (agent._conf.logUncaughtExceptions === true) {
+    if (agent._conf && agent._conf.logUncaughtExceptions === true) {
```

Comment:
REVIEW NOTE: This is a guard against calling agent.handleUncaughtExceptions() and then capturing an uncaught exception before agent.start()